packages/frontend/core/src/components/affine/setting-modal/general-setting/index.tsx

```diff
@@ -32,6 +35,10 @@ export const useGeneralSettingList = (): GeneralSettingList => {
   const hasPaymentFeature = useLiveData(
     serverConfig.features$.map(f => f?.payment)
   );
+  const userFeatureService = useService(UserFeatureService);
```

Comment:
Because `UserFeatureService` does not automatically load data, I suggest adding a `revalidate()` call here.

```ts
useEffect(() => {
  userFeatureService.userFeature.revalidate();
}, [userFeatureService]);
```

PS: `ServerConfigService` and `AuthService` are not needed, as they always automatically load data.

https://github.com/toeverything/AFFiNE/blob/35ce4adffeacc76e6acc0effc7b05043b742ecb7/packages/frontend/core/src/components/affine/setting-modal/setting-sidebar/index.tsx#L274-L276

Reply:
Got it! Thanks for your comments, it helps a lot.